app/assets/javascripts/oxalis/model.js

```diff
@@ -443,6 +446,7 @@ export class OxalisModel {
         depth: tracing.boundingBox.depth,
       },
       resolutions,
+      maxZoomStep: resolutions.length - 1,
```

Comment:
The `tracingLayer` is pushed into `layers` later, which is then mapped using the `adaptResolutionInfoForLayer`. To me it looks like this assignment of `maxZoomStep` is redundant as the `adaptResolutionInfoForLayer` function will do it later anyway.

Reply:
Good catch! However, flow complains if I omit this property. I'd like to refactor how layer handling is done, anyway. So, I think I'll leave it as is for now.